homeassistant/components/epsonworkforceprinter/sensor.py

```diff
@@ -0,0 +1,85 @@
+"""Support for Epson Workforce Printer."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_HOST, CONF_MONITORED_CONDITIONS
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['epsonprinter==0.0.8']
+
+_LOGGER = logging.getLogger(__name__)
+MONITORED_CONDITIONS = {
+    'black': ['Inklevel Black', '%', 'mdi:water'],
+    'magenta': ['Inklevel Magenta', '%', 'mdi:water'],
+    'cyan': ['Inklevel Cyan', '%', 'mdi:water'],
+    'yellow': ['Inklevel Yellow', '%', 'mdi:water'],
+    'clean': ['Inklevel Cleaning', '%', 'mdi:water'],
+}
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_MONITORED_CONDITIONS):
+        vol.All(cv.ensure_list, [vol.In(MONITORED_CONDITIONS)]),
+})
+SCAN_INTERVAL = timedelta(minutes=60)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the cartridge sensor."""
+    host = config.get(CONF_HOST)
+
+    from epsonprinter_pkg.epsonprinterapi import EpsonPrinterAPI
+    api = EpsonPrinterAPI(host)
+    if not api.available:
+        raise PlatformNotReady("Printer is not available")
```

Comment:
There is no need for the explanation string.

Reply:
Okay, assumed it was required, removed it